src/commonMain/kotlin/exchange.dydx.abacus/state/v2/supervisor/SubaccountSupervisor.kt

```diff
@@ -95,6 +64,26 @@ internal class SubaccountSupervisor(
     private val accountAddress: String,
     internal val subaccountNumber: Int,
 ) : DynamicNetworkSupervisor(stateMachine, helper, analyticsUtils) {
+
+    private val payloadProvider: SubaccountTransactionPayloadProviderProtocol = SubaccountTransactionPayloadProvider(
+        stateMachine = stateMachine,
+        subaccountNumber = subaccountNumber,
+        helper = helper,
+        accountAddress = accountAddress,
+    )
+    private val transactionSupervisor = SubaccountTransactionSupervisor(
```

Comment:
probably want to pass into constructor for better testability

Reply:
Yes, we can do that when adding unit tests to the class.